### PR TITLE
#77 - Fixes theme & breakpoints exports

### PR DESCRIPTION
**Pull request by srobfr**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-78.vercel.app'>https://matou-technologies-purrfect-ui-storybook-78.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/43cQPA8T3mUktHPrzdnx4qgwDch6'>https://vercel.com/matou-technologies/purrfect-ui/43cQPA8T3mUktHPrzdnx4qgwDch6</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/index.ts [hunk 1]
```diff
@@ -67,14 +67,4 @@ export type { TextFieldProps } from "src/form/TextField/TextField";
...
-  heading2,
-  heading3,
-  heading4,
-  heading5,
-  heading6,
-  body1,
-  body2,
-  narrow,
-} from "src/guidelines/theme";
+export * from "src/guidelines/theme";
```
Comment (MarcoPOtard) [inline src/index.ts:70]: There is something I missing.
In src/guidelines/theme/index.ts, you export typographies
Here you export everything from theme
But in files like src/navigation/Breadcrumb/Breadcrumb.tsx you import body from typographies file and not from theme/index file
I don't understand something or I missed something ?

File: Makefile [hunk 2]
```diff
@@ -40,3 +43,7 @@ certs:
 	@mkcert -key-file ./docker/traefik/certs/key.pem -cert-file ./docker/traefik/certs/cert.pem \
 		'purrfect-ui.localhost' \
 		'purrfect-ui.traefik.localhost'
+
+help: ## Outputs this help screen
+	@grep -E '(^[a-zA-Z0-9\./_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}{printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
+.DEFAULT_GOAL := help
```
Comment (MarcoPOtard) [inline Makefile:49]: I understand the pre-commit, but I don't understand this one. Could you explain me please ?

Comment (srobfr) [reply Makefile:49]: This is a default target which lists the available targets when you call `make` without specifying a target.

File: src/index.ts (hunk 1 above)
Comment (srobfr) [reply src/index.ts:70]: Both would work, actually.
I think the index files are mainly used for exporting (to the project using purrfect-ui), while you still can use direct imports from within purrfect-ui

Comment (MarcoPOtard) [reply src/index.ts:70]: ok

File: Makefile (hunk 2 above)
Comment (CJCCyril) [reply Makefile:49]: Maybe replace `ci:` content by pre-commit one ? Also, nothing but pnpm and help will be shown cause of the missing comments ?

**Review by CJCCyril — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (srobfr) [reply Makefile:49]: It would be a bad idea, as format:check --write can actually apply changes to the code.
I'll add the missing docs.

**Pull request merged**